<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1369</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PLSpec 0.01 beta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="layout_main_v">
      <item>
       <widget class="QTabWidget" name="tab_main_plots">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_spec_corrected">
         <attribute name="title">
          <string>Spectrum corrected</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="layout_spec_raw_v">
            <item>
             <widget class="PlotWidget" name="plot_main_corrected" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_spec_raw">
         <attribute name="title">
          <string>Spectrum raw</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="layout_spec_corrected_v">
            <item>
             <widget class="PlotWidget" name="plot_main_raw" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_panel_h">
        <item>
         <widget class="QGroupBox" name="gbox_mono">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>250</height>
           </size>
          </property>
          <property name="title">
           <string>Mono</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QGridLayout" name="layout_mono_gbox_g">
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="sbox_slitin"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Slit in (mm)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="sbox_position"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Position (nm)</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Slit out (mm)</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="sbox_slitout"/>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btn_apply">
             <property name="text">
              <string>Apply</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbox_daq">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>250</height>
           </size>
          </property>
          <property name="title">
           <string>DAQ</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Integration time (s)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="sbox_integration_time"/>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="btn_cont">
             <property name="text">
              <string>Contonuous...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbox_spectra">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>250</height>
           </size>
          </property>
          <property name="title">
           <string>Spectra</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="layout_spectra_h">
             <item>
              <layout class="QVBoxLayout" name="layout_spec_left_v">
               <item>
                <widget class="QPushButton" name="btn_add_scan">
                 <property name="text">
                  <string>Add Scan</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_redo_scan">
                 <property name="text">
                  <string>Redo Scan...</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btn_start_scan">
                 <property name="text">
                  <string>Start Scan</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_abort_scan">
                 <property name="text">
                  <string>Abort Scan</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="table_specs">
               <property name="minimumSize">
                <size>
                 <width>820</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>1200</width>
                 <height>400</height>
                </size>
               </property>
               <row>
                <property name="text">
                 <string>1</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>tag</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>start (nm)</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>stop (nm)</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>step (nm)</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>slit (mm)</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>T (K)</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>scale</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>shift (nm)</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="layout_spec_right_v">
               <item>
                <widget class="QPushButton" name="btn_update">
                 <property name="text">
                  <string>Update</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_merge">
                 <property name="text">
                  <string>Merge</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_delete">
                 <property name="text">
                  <string>Delete</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_save">
                 <property name="text">
                  <string>Save...</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1369</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
